_source/_posts/2018-05-18-node-authentication-with-passport-and-oidc.md

```diff
@@ -47,47 +61,47 @@ npm install express-generator -g
 Then use the `express` command to scaffold a base Node and Express application.
 
 ```bash
-express -e --git passport-oidc-example
+express -e --git okta-node-passport-oidc-example
 ```
 
 The generator will quickly create a new app in the **passport-oidc-example** folder. It uses the Embedded JavaScript syntax for the view templates and will generate a base **.gitignore** file. There will be instructions at the bottom of the output telling you how to proceed.
 
 ```bash
 change directory:
-  $ cd passport-oidc-example
+  $ cd okta-node-passport-oidc-example
 
 install dependencies:
   $ npm install
 
 run the app:
-  $ DEBUG=passport-oidc-example:* npm start
+  $ DEBUG=okta-node-passport-oidc-example:* npm start
 ```
 
-Go ahead and change into the new directory, and install the dependencies. I use [Visual Studio Code](https://code.visualstudio.com/) for my Node development which has great support for writing and debugging Node applications. It works on all platforms and is completely free. Running the application with a debugger attached is as easy as hitting the `F5` key!
+Go ahead and change into the new directory and install the dependencies. I use [Visual Studio Code](https://code.visualstudio.com/) for my Node development which has excellent support for writing and debugging Node applications. It works on all platforms and is completely free. Running the application with a debugger attached is as easy as hitting the `F5` key!
 
-Once you have VS Code installed, you can open the project from the command line using the `code` command.
+Once VS Code is installed, you can open the project from the command line using the `code` command.
```

Comment:
```suggestion
Once you've installed VS Code, you can open the project from the command line using the `code` command.
```
if you're going to change this sentence, I'd recommend going from passive to active voice instead of adjusting the verb choice.